client/src/components/Content.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { FaBeer } from 'react-icons/fa';
+import { FaHeart } from 'react-icons/fa';
+import Photo from './Photo.jsx';
+
+const Content = (props) => {
+  return (
```

Comment:
Please use an implicit return here. Example:

```
const Content = (props) => (
  ...JSX Here...
);
```

Notice there is no need for the `return` keyword here.